pubtools/iib/iib_ops.py

```diff
@@ -131,7 +131,7 @@
 ADD_CMD_ARGS[("--bundle",)] = {
     "group": "IIB service",
     "help": "<hostname>/<namespace>/<image>:<tag> of bundle",
-    "required": True,
+    "required": False,
```

Comment:
Can we add a default value here please? An empty list

Reply:
It's None by default. Is there a reason for changing it?

Reply:
Yes, because IIB expects the `bundles` parameter to be an empty array or an array of strings.

Reply:
Well, I just looked at the IIB code and it looks like it will default to an empty list if the parameter is not provided. So I guess it should be fine.